docs/spec.yml

```diff
@@ -315,6 +565,25 @@ components:
           scopes:
             read: Grants read access
             write: Grants write access
+  schemas:
+    techRecordGETComplete:
```

Comment:
why aren't there any testable and skeleton schemas?

Reply:
Good shout, will add them